server/models/User.js

```diff
@@ -0,0 +1,11 @@
+const {Schema, model} = require("mongoose");
+
+const User = new Schema({
+    email: {type: String, required: true,  trim: true, unique: true},
+    password: {type: String, required: true},
+    verified: {type: Boolean},
+    verificationLink: {type: String},
```

Comment:
Maybe we don't need to store the verification link.
Because it will be in user's email.

Reply:
verification link is needed to redirect user to verify account by email

Reply:
I've implemented it before but I didn't need it as I remember and maybe check this tutorial for that.
[https://dev.to/cyberwolve/how-to-verify-user-email-in-node-express-eaj](url)

Reply:
the link is not working
i took another reference anyway. lemme go in the way that i started. if it's not a good option i'll just delete it later on

Reply:
Alright. I will merge this PR